cost-analyzer/values.yaml

```diff
@@ -221,3 +221,25 @@ grafana:
 
 serviceAccount:
   create: true
+
+# These configs are settable from either the /settings.html page or value.yaml
+# This block overrides set configs on /settings.html on pod restart
+#kubecostProductConfigs: 
+#  defaultModelPricing:
```

Comment:
is it worth adding a comments above this block? proposal is below.

`# default (monthly) model prices are used predominately for on-prem clusters`